index.html

```diff
@@ -51,7 +51,206 @@ <h1 class="first">I'm Recillah Glad To See You!</h1>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="linkedin">
+                                <img src="Images/Linkedin-icon.svg" alt="linkedin">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="medium">
+                                <img src="Images/M-Vector.svg" alt="medium">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="github">
+                                <img src="Images/C-Vector.svg" alt="github">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="angelist"
+                            ><img src="Images/P-Vector.svg" alt="angelist">
+                        </a>
+                        </li>
+                    </ul>
+                    <button>Get my resume</button>
+                </div>
+            </div>
+            
+            <div class="skill-list">
+                <ul>
+                    <li class="frame-12">
+                        <div class="frame-7">
+                            <h4 class="languages">Languages</h4>
+                            <a class="arrow-down" href="#"><img src="Images/arrow-down-icon.png" alt="Arrow-Down Icon"></a>
+                        </div>
+                        <ul class="language-list">
+                            <li class="content">
+                                <img class="icon" src="Images/Ellipse-1.png" alt="JavaSript-logo">
+                                <p class="text">JavaScript</p>
+                            </li>
+                            <li class="content">
+                                <img class="icon" src="images/HTML-logo.png" alt="html-logo">
+                                <p class="text">HTML</p>
+                            </li>
+                            <li class="content">
+                                <img class="icon" src="Images/CSS-logo.png" alt="css-logo">
+                                <p class="text">CSS</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li>
+                        <div class="frame-7">
+                            <h4 class="languages">Frameworks</h4>
+                            <a class="arrow-down" href="#"><img src="Images/arrow-right.png" alt="Arrow Right Icon">
+                            </a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="frame-7">
+                            <h4 class="languages">Skills</h4>
+                            <a class="arrow-down" href="#"><img src="Images/arrow-right.png" alt="Arrow Right Icon"></a>
+                        </div>
+                    </li>
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, remove these bullets from your design to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 👌

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177019677-05b7524d-3910-4784-bf8d-51867cd9dbcf.PNG" width="100%" />  | <img src="https://user-images.githubusercontent.com/45231276/177019670-5062a589-22f2-4a03-81ed-9b4996755d33.PNG" width="100%" /> |